### DISCUSSION TITLE
How can we change the shape of the standard link anchor default shapes from arrow to circle or square or  any custom shape

### DISCUSSION THREAD

**Discussion by vyombriq** [clientIO/joint#2800] [Q&A] [answered]
### Introduction

I want o change the default anchor shapes to custom shapes , can you suggest me how can we do that , you can see the default highlighted anchor shapes.

Note : I am using standard Link and want to continue using that 
![image](https://github.com/user-attachments/assets/3c93197c-e8cd-4780-bbaa-cf6301249ce2)

### Restrictions & Constraints

 I am using standard Link and want to continue using that 

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment, marked as answer]
If you need to change the arrowheads, you can do it by using [`sourceMarker`](https://docs.jointjs.com/api/dia/attributes/#sourcemarker) /  [`targetMarker`](https://docs.jointjs.com/api/dia/attributes/#targetmarker) attributes.

To see examples of custom arrowheads visit: https://www.jointjs.com/demos/connector-arrows
